tests/conformance/conformance_suite_test.go

```diff
@@ -341,14 +341,29 @@ func CompareJSON(a, b map[string]interface{}, skip []string) ([]string, []string
 			continue
 		}
 		if reflect.TypeOf(v) != reflect.TypeOf(vb) {
+			if (reflect.TypeOf(v) == nil && reflect.ValueOf(vb).Len() == 0) {
+				continue
+			} else if (reflect.TypeOf(vb) == nil && reflect.ValueOf(v).Len() == 0) {
```

Comment:
No need for else here.

Reply:
Hi @rhatdan 
As currently we call this functions like CompareJSON(dockerimage, buildahimage, skip) we don't need else here. But if we consider this functions as common use function, we don't know everyone will call it with such order. So I'd like keep this here to cover all kinds of situation.

Reply:
sorry I wasn't very clear.  All I meant is to remove the word `else`  Since the first block executes a continue. it will not execute the the second block.  
I find `else if` harder to read and don't want them when they are not necessary.
``
		if reflect.TypeOf(v) != reflect.TypeOf(vb) {
			if (reflect.TypeOf(v) == nil && reflect.ValueOf(vb).Len() == 0) {
				continue
			} 
                        if (reflect.TypeOf(vb) == nil && reflect.ValueOf(v).Len() == 0) {
```

Reply:
Oh. Thanks a lot for your explanation. Get the point now. Will format a patch to fix this.